docs/Clearwater_Configuration_Options_Reference.md

```diff
@@ -119,9 +119,11 @@ This section describes optional configuration options, particularly for ensuring
     * `pcscf,icscf` - a Record-Route header is added just after requests come from or go to a P-CSCF or I-CSCF - that is, at the start and end of originating handling and the start and end of terminating handling
     * `pcscf,icscf,as` - a Record-Route header is added after requests come from or go to a P-CSCF, I-CSCF or application server - that is, at the start and end of originating handling, the start and end of terminating handling, and between each application server invoked
 * `force_hss_peer` - when set to an IP address or hostname, Homestead will create a connection to the HSS using this value, but will still use the `hss_realm` and `hss_hostname` settings for the Destination-Host and Destination-Realm Diameter AVPs. This is useful when your HSS's Diameter configuration does not match the DNS records.
-* `hss_mar_lowercase_unknown` - some Home Subscriber Servers (particularly old releases of OpenIMSCore HSS) expect the string 'unknown' rather than 'Unknown' in Multimedia-Auth-Requests when Clearwater cannot tell what authentication type is expected. Setting this option to 'Y' will make Homestead send requests in this format.
-* `hss_mar_force_digest` - if Clearwater cannot tell what authentication type a subscriber is trying to use, this forces it to assume 'SIP Digest' and report that in the Multimedia-Auth-Request, rather than 'Unknown'.
-* `hss_mar_force_aka` - if Clearwater cannot tell what authentication type a subscriber is trying to use, this forces it to assume 'Digest-AKA-v1' and report that in the Multimedia-Auth-Request, rather than 'Unknown'.
+* `hss_mar_scheme_unknown` - if Clearwater cannot tell what authentication type a subscriber is trying to use, this field determines what authentication scheme it requests in the Multimedia-Auth-Request. Default value is 'Unknown'.
+* `hss_mar_scheme_digest` - if Clearwater determines a subscriber is trying to use password-based digest authentication, this field determines what authentication scheme it requests in the Multimedia-Auth-Request. Default value is 'SIP Digest'.
+* `hss_mar_scheme_akav1` - if Clearwater determines a subscriber is trying to use AKAv1 authentication, this field determines what authentication scheme it requests in the Multimedia-Auth-Request. Default value is 'Digest-AKAv1-MD5'.
+* `hss_mar_scheme_akav2` - if Clearwater determines a subscriber is trying to use AKAv2 authentication, this field determines what authentication scheme it requests in the Multimedia-Auth-Request. Default value is 'Digest-AKAv2-SHA-256'.
+* `allow_fallback_ifcs` - when Clearwater cannot find an Identity element in the user profile XML from the HSS that matches the served user, it will normally not invoke any services. When this is set to Y, it will use the first set of Initial Filter Criteria in the profile (giving the ability to work around HSSes that return something unexpected, or the ability to get basic support for wildcard public service identities).
```

Comment:
I thought this was covered by the first sentence ("it will normally not invoke any services") but I've reworded to make the contrast clearer ("it will _by default_ not invoke any services. When this is set to Y, it will _instead_ use the first set...")